src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -183,15 +183,16 @@
     private int buildScheduleDelay;
     private int dynamicConfigRefreshInterval;
     private boolean loadProjectListOnStartup;
-    private int projectListRefreshInterval;
+    @Deprecated
+    private transient int projectListRefreshInterval;
```

Comment:
I think you should add to the readResolve method a conversion from ```projectListRefreshInterval``` to ```projectListFetchDelay```. Unless this is the new field that @GLundh added in his previous pr? In that case you don't need to deprecate and make transient because it has never been in a released version of the plugin IIRC.